opencti-platform/opencti-front/src/components/GradientButton.tsx

```diff
@@ -0,0 +1,48 @@
+import { styled } from '@mui/material/styles';
+import Button, { ButtonProps } from '@mui/material/Button';
+
+interface GradientButtonProps extends ButtonProps {
+  lightModeStartColor?: string;
+  lightModeEndColor?: string;
+  darkModeStartColor?: string;
+  darkModeEndColor?: string;
+  withShadow?: boolean;
```

Comment:
I made it for the example in Filigran-UI, we need to choose only one in OCTI and remove that boolean

Reply:
Done